zerver/views/home.py

```diff
@@ -19,7 +19,7 @@
 from zerver.lib.user_counts import realm_user_count
 from zerver.lib.utils import statsd
 from zerver.models import PreregistrationUser, Realm, Stream, UserProfile
-from zerver.views.portico import hello_view
+from corporate.views.portico import hello_view
```

Comment:
This `import` needs to be conditional, since `corporate/` is excluded from release tarballs. And, as pointed out by CI, it needs to be `isort`ed.

Reply:
Ok, fixed!